orchagent/portsorch.cpp

```diff
@@ -3166,6 +3171,46 @@ void PortsOrch::doTask(NotificationConsumer &consumer)
         }
 
         sai_deserialize_free_port_oper_status_ntf(count, portoperstatus);
+    } else if (op == "media_change") {
```

Comment:
media settings should go to standard port app db, like the admin_status in the port. I do not understand why choose the notification consumer here. 

Please make sure your code style is consistent, you need to keep a space for after if.

Reply:
Hi Guohan,
I feel that media event is analogous to oper state notification generated by ASIC. These are from South bound APIs. If we see the oper status implementation, a notification consumer is used which is from syncd to orchagent. I used the same logic to implement this as similar to oper status. On other hand I feel admin status is application triggerd change from user. 
Please let me know your views. If i still want to go ahead and implement it similar to admin status I will go ahead and update the PR.